chart/skywalking/templates/es-init.job.yaml

```diff
@@ -15,7 +15,7 @@
 
 # https://docs.sentry.io/server/installation/docker/#running-migrations
 
-{{- if hasPrefix "elasticsearch" .Values.oap.storageType }}
```

Comment:
This was to match `elasticsearch` and `elasticsearch7`, we can modify it to equals since we merge ES versions but modifying yo suffix is wrong in any point of view.

Reply:
> This was to match `elasticsearch` and `elasticsearch7`, we can modify it to equals since we merge ES versions but modifying yo suffix is wrong in any point of view. 

Ok I just read the related issue and notice we never take care of the Zipkin-elesticsearch...

Reply:
> > This was to match `elasticsearch` and `elasticsearch7`, we can modify it to equals since we merge ES versions but modifying yo suffix is wrong in any point of view. 
> 
> Ok I just read the related issue and notice we never take care of the Zipkin-elesticsearch...

What about changing to `contains`? So that we don't break the compatibility when users install OAP back to the versions where we have elasticsear7?

Reply:
Also, just a tip, zipkin-skywalking option is going to be removed in 9.1.0.

Recommend to follow Zhenxu way, keep max old versions compatibility.